plugin/pkg/global/resourcereferencemanager/admission_test.go

```diff
@@ -194,6 +225,15 @@ var _ = Describe("resourcereferencemanager", func() {
 			fakeAuthorizer = fakeAuthorizerType{}
 			admissionHandler.SetAuthorizer(fakeAuthorizer)
 
+			discoveryClient = &discoveryfake.FakeDiscovery{Fake: &testing.Fake{Resources: discoveryClientResources}}
+			admissionHandler.SetDiscoveryClient(discoveryClient)
+
+			scheme = runtime.NewScheme()
+			_ = corev1.AddToScheme(scheme)
```

Comment:
Please don't ignore errors in tests:
```suggestion
			Expect(corev1.AddToScheme(scheme)).To(Succeed())
```